client/js/components/user/CustomerSettings/CustomerSettingsSupport.vue

```diff
@@ -41,14 +41,14 @@
             autocomplete="tel"
             class="u-mb-0_75"
             data-cy="phoneNumber"
-            :data-dp-validate-error="customerContact.eMailAddress === ''
-              ? 'error.phone_or_email.required'
+            :data-dp-validate-error="customerContact.phoneNumber === ''
+              ? 'error.phone.required'
               : 'error.phone.pattern'"
```

Comment:
```suggestion
           :data-dp-validate-error="!customerContact.phoneNumber ? 'error.phone.required' : 'error.phone.pattern'"
```
If you write it like that it is readable enough to put it in one line which would improve the overall readability imho.

Reply:
done https://github.com/demos-europe/demosplan-core/pull/2156/commits/8a2b9fbbb0a9234ca82acf496b764afa254f9c69

Reply:
ah, I seedone https://github.com/demos-europe/demosplan-core/pull/2156/commits/4e5a64bd4000ddeb2b234fbe8925ab94b191600a